src/components/implementation/no_interface/scheddev/fprr/sl_mod_fprr.c

```diff
@@ -35,6 +38,7 @@ void
 sl_mod_block(struct sl_thd_policy *t)
 {
 	ps_list_rem_d(t);
+	sl_timeout_mod_block(sl_mod_thd_get(t), 0);
```

Comment:
Do we always want to set the timer when we block a thread?   If so, I don't see why this would be in scheduling-specific policy code.